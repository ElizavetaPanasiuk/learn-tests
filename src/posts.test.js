import Posts from './Posts';
import { render, configure, shallow, mount } from 'enzyme';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';

configure({
  adapter: new Adapter()
});

describe('<Posts />', () => {
  const component = shallow(<Posts />);
  let wrapper;

  it('should render include a link', () => {
    wrapper = component.find('.newsList');
    expect(wrapper.length).toBeGreaterThanOrEqual(1);
  });

  it('renders posts', () => {
    console.log(component);
    expect(component).toMatchSnapshot();
  });
});