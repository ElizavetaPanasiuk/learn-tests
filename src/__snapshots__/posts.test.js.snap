// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Posts /> renders posts 1`] = `ShallowWrapper {}`;
